includes/class-wcpdf-settings-debug.php

```diff
@@ -54,7 +54,7 @@ public function debug_tools( $tab, $section ) {
 						$old_path = WPO_WCPDF()->main->get_tmp_base( false );
 						$new_path = WPO_WCPDF()->main->get_tmp_base();
 						WPO_WCPDF()->main->copy_directory( $old_path, $new_path );
-						printf('<div class="notice notice-success"><p>%s</p></div>', __( 'Random string created!', 'woocommerce-pdf-invoices-packing-slips' ) );
+						sprintf( __( 'Temporary folder moved to `%s`', 'woocommerce-pdf-invoices-packing-slips' ), $new_path );
```

Comment:
you removed too much :octocat:  only `__()` should have been replaced by the `sprintf()`. Now there is no notice at all anymore.